src/commands/server/logs.ts

```diff
@@ -0,0 +1,62 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command, flags } from '@oclif/command'
+import { string } from '@oclif/parser/lib/flags'
+import * as Listr from 'listr'
+import * as notifier from 'node-notifier'
+import * as path from 'path'
+
+import { cheDeployment, cheNamespace, listrRenderer } from '../../common-flags'
+import { CheTasks } from '../../tasks/che'
+import { K8sTasks } from '../../tasks/platforms/k8s'
+
+export default class Logs extends Command {
+  static description = 'Retrieve Eclipse Che logs'
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    chenamespace: cheNamespace,
+    'listr-renderer': listrRenderer,
+    'deployment-name': cheDeployment,
+    directory: string({
```

Comment:
Maybe it is better to be consistent and if we use quotes for some fields, let's use them in whole json?

Reply:
Not possible due to warning
`Unnecessarily quoted property 'directory' found. (object-literal-key-quotes)tslint(1)`